www/web-proxy-sso/src/opnsense/mvc/app/controllers/OPNsense/ProxySSO/forms/general.xml

```diff
@@ -11,4 +11,10 @@
 		<type>dropdown</type>
 		<help>Select Windows Server version for AD controller</help>
 	</field>
+	<field>
+		<id>ProxySSO.KerberosHostName</id>
+		<label>This host Kerberos account name in AD</label>
```

Comment:
`Kerberos Account Name Of This Host In AD` would sound better. I let it open to you if you would like to change it.